components/post_view/reaction_list/index.js

```diff
@@ -5,30 +5,33 @@ import {connect} from 'react-redux';
 import {bindActionCreators} from 'redux';
 
 import {getChannel} from 'mattermost-redux/selectors/entities/channels';
-import {makeGetReactionsForPost} from 'mattermost-redux/selectors/entities/posts';
+import {getReactionsForPosts} from 'mattermost-redux/selectors/entities/posts';
 import {getConfig} from 'mattermost-redux/selectors/entities/general';
 
 import {addReaction} from 'actions/post_actions.jsx';
 import {scrollPostList} from 'actions/views/channel';
 
 import ReactionList from './reaction_list.jsx';
 
-function makeMapStateToProps() {
-    const getReactionsForPost = makeGetReactionsForPost();
+function makeMapStateToProps(state, ownProps) {
+    // const getReactionsForPost = makeGetReactionsForPost();
 
-    return function mapStateToProps(state, ownProps) {
-        const config = getConfig(state);
-        const enableEmojiPicker = config.EnableEmojiPicker === 'true' && !ownProps.isReadOnly;
+    // return function mapStateToProps(state, ownProps) {
+    const config = getConfig(state);
+    const enableEmojiPicker = config.EnableEmojiPicker === 'true' && !ownProps.isReadOnly;
 
-        const channel = getChannel(state, ownProps.post.channel_id) || {};
-        const teamId = channel.team_id;
+    const channel = getChannel(state, ownProps.post.channel_id) || {};
+    const teamId = channel.team_id;
 
-        return {
-            teamId,
-            reactions: getReactionsForPost(state, ownProps.post.id),
-            enableEmojiPicker,
-        };
+    // console.log('ReactionList::mapStateToProps', ownProps.post.id,
```

Comment:
Commented out lines?

Reply:
Gah, I have no idea why this version ended up here. Fixed!